daisy_workflows/image_import/debian/translate_debian.sh

```diff
@@ -64,9 +64,9 @@ EOF
     apt-get install --assume-yes --no-install-recommends \
     google-cloud-packages-archive-keyring \
     google-cloud-sdk \
-    google-compute-engine-${DEB_RELEASE} \
-    google-compute-engine-init-${DEB_RELEASE} \
-    google-config-${DEB_RELEASE}
+    google-compute-engine \
+    python-google-compute-engine \
+    pythone3-google-compute-engine
```

Comment:
python3?

Reply:
You say potato- I say potatoe, but you will still be right ;) Fixed.